bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
Sorry, by breaking an upper bound I mean that `dev` ends up violating a `< 4.0.0` version constraint for example, preventing pinning against a git repo.